flow/ProtocolVersion.h

```diff
@@ -130,6 +130,7 @@ class ProtocolVersion {
 	PROTOCOL_VERSION_FEATURE(0x0FDB00B063010000LL, SmallEndpoints);
 	PROTOCOL_VERSION_FEATURE(0x0FDB00B063010000LL, CacheRole);
 	PROTOCOL_VERSION_FEATURE(0x0FDB00B070010000LL, StableInterfaces);
+	PROTOCOL_VERSION_FEATURE(0x0FDB00B070010000LL, PartitionTransaction);
```

Comment:
I think we should bump this to `0x0FDB00B071010000LL`. since release-7.0 just being cut today, this feature won't make into 7.0 version.

But I guess this change may only be possible after my PR #4752 being merged. So, maybe add a TODO here so we won't forget this.

Reply:
I see the PR is already merged, I will make this change.